_posts/2023-03-09-solidity-developer-survey-2022-results.md

```diff
@@ -0,0 +1,452 @@
+---
+layout: post
+published: true
+title: 'Solidity Developer Survey 2022 Results'
+date: '2023-03-09'
+author: Franziska Heintel
+category: Announcements
+---
+
+In this post, we will be summarizing and analyzing the results of the [2022 Solidity Developer Survey](https://blog.soliditylang.org/2022/12/07/solidity-developer-survey-2022-announcement/).
+
+First of all, a big thank you to everybody who took the time and participated and to everybody who helped us spread the word about it!
+
+This year, we received a smashing 1401 responses. That is more than a 3x in responses compared to the previous survey and we couldn't be happier with the turnout.
+
+Your input is invaluable to us and plays a crucial role in helping to continuously improve the Solidity developer experience as a whole.
```

Comment:
```suggestion
In this post, we will summarize and analyze the results of the [2022 Solidity Developer Survey](https://blog.soliditylang.org/2022/12/07/solidity-developer-survey-2022-announcement/).

First of all, a big thank you to everybody who took the time and participated and to everybody who helped us spread the word about it!

This year, we received a smashing 1401 responses. That is more than a 3x in responses compared to the previous survey and we couldn't be happier with the turnout.

Your input is invaluable to us and plays a crucial role in helping to continuously improve the Solidity developer experience as a whole.
```

Reply:
Since this is what gets scrapped by Google I would, move what is it a bit into the first words of the blogspot like so:

```suggestion
The [2022 Solidity Developer Survey](https://blog.soliditylang.org/2022/12/07/solidity-developer-survey-2022-announcement/) results are in! In this post, we will be summarizing and analyzing them.

First of all, a big thank you to everybody who took the time and participated and to everybody who helped us spread the word about it!

This year, we received a smashing 1401 responses. That is more than a 3x in responses compared to the previous survey and we couldn't be happier with the turnout.

Your input is invaluable to us and plays a crucial role in helping to continuously improve the Solidity developer experience as a whole.
```